code/modules/antagonists/eldritch_cult/eldritch_antag.dm

```diff
@@ -216,6 +235,25 @@
 /datum/antagonist/heretic/proc/get_all_knowledge()
 	return researched_knowledge
 
+/////////////
+// Economy //
+/////////////
+
+/datum/antagonist/heretic/proc/gain_favor(points,dread = FALSE)
+	favor_earned+=points
+	if (dread)
```

Comment:
```suggestion
/datum/antagonist/heretic/proc/gain_favor(points,_dread = FALSE)
	favor_earned+=points
	if (_dread)
```

Reply:
Thanks for pointing it out. Must have slipped me :D